From 2882a626df3dafcd6437db39544f612bd079fd4e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Daniel=20M=C3=BCller?= <deso@posteo.net>
Date: Sat, 25 May 2024 07:22:58 -0700
Subject: [PATCH] Work around Unicode character display issues

Version 3.4 of the program broke the display of certain Unicode
characters [0] [1] and it is unclear whether this will ever get
addressed. Work around the issue for the time being.

[0] https://github.com/tmux/tmux/issues/3865
[1] https://github.com/tmux/tmux/issues/3923
---
 screen-write.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/screen-write.c b/screen-write.c
index 6892d0..1174cb 100644
--- a/screen-write.c
+++ b/screen-write.c
@@ -2088,7 +2088,7 @@ screen_write_combine(struct screen_write_ctx *ctx, const struct grid_cell *gc)
 	if (utf8_is_zwj(ud))
 		zero_width = 1;
 	else if (utf8_is_vs(ud))
-		zero_width = force_wide = 1;
+		zero_width = 1;
 	else if (ud->width == 0)
 		zero_width = 1;
 
-- 
2.44.1

